assets/scss/00-base/_fonts-langs-import.scss

```diff
@@ -0,0 +1,152 @@
+@forward "paths";
+@forward "variables";
+
+@use "paths" as *;
+@use "variables" as *;
+// Variable sans fonts
+$adlam:						U+1E900-1E95F, U+2E41; // Uses reversed comma
```

Comment:
Make sure to not use any of these variables globally in your `@use` statement (do not use `as *`).